system/ee/EllisLab/ExpressionEngine/Core/Core.php

```diff
@@ -289,7 +289,7 @@ protected function getErrorRouting()
 	 */
 	public function setTimeLimit($t)
 	{
-		if (function_exists("set_time_limit") && @ini_get("safe_mode") == 0)
```

Comment:
According to PHP docs https://www.php.net/manual/en/function.set-time-limit.php `set_time_limit` has no effect with safe mode on. So this line should be here